ci-operator/step-registry/upi/install/libvirt/upi-install-libvirt-commands.sh

```diff
@@ -93,7 +103,6 @@ fi
 cp ${SHARED_DIR}/install-config.yaml /tmp
 ${OCPINSTALL} create ignition-configs --dir /tmp
 
-save_credentials
```

Comment:
I cannot remember my reason for saving credentials here, but if we can get a clean rehearsal with this removed, I'll consider that reason to be "irrelevant", ha ha.

Reply:
This is required i think, when i tried rehearsing job by removing this, it was failing with below error
`level=error msg=Cluster operator authentication Degraded is True with IngressStateEndpoints_MissingSubsets::OAuthClientsController_SyncError::OAuthServerDeployment_PreconditionNotFulfilled::OAuthServerRouteEndpointAccessibleController_SyncError::OAuthServerServiceEndpointAccessibleController_SyncError::OAuthServerServiceEndpointsEndpointAccessibleController_SyncError::WellKnownReadyController_SyncError: IngressStateEndpointsDegraded: No subsets found for the endpoints of oauth-server`

when i added it back [here](https://github.com/openshift/release/blob/e1b7a9ab7a686bb16f03323ff1dc54cd6cdf79da/ci-operator/step-registry/upi/install/libvirt/upi-install-libvirt-commands.sh#L43), it started passing. Thanks for pointing this out.